src/components/video-player/VideoPlayer.styles.ts

```diff
@@ -0,0 +1,11 @@
+export const styles = {
+  playerWrapper: {
+    position: 'relative',
+    paddingTop: '56.25%'
```

Comment:
we use short convention naming `pt`
I'm suggest round down/up `56.25%` what do you think ?

Reply:
we can't give it up.
because we need to keep the aspect ratio 16:9.
Let's take a 1280 x 720 player as an example.
720/1280 = 0.5625
0.5625 * 100 = 56.25

Reply:
could we use [aspect ratio](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) property instead?

Reply:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/101183408/292a2513-8297-4b33-92ad-fe522a49f694)
fixed

Reply:
Cool! It looks cleaner now